### ISSUE TITLE
Replace current favicon with a less scary favicon

### ISSUE THREAD

**Issue by Erioldoesdesign** [foss-responders/fossresponders.com#19]
The current mask-wearing emoji is not very nice! 

![Screenshot 2020-03-19 at 17 55 33](https://user-images.githubusercontent.com/11681324/77098812-e0165a80-6a0a-11ea-8be2-d582e129a955.png)

Let's do a better favicon that follows on our branding once decided. 

Blocked by #10

**Comment by Erioldoesdesign** [comment]
Okay @RichardLitt here are two Favicon options...I have a feeling the single figure will work best at 16x16px...which is tricky because the two figures work at bigger sizes like 32x32px! argh!

https://drive.google.com/open?id=10ZI4FgZfh1h2LTxwboNPQN0hyb7_PSi9

Can you see what you think is best and go with that?

**Comment by RichardLitt** [comment]
I think we can use the one-figure for the smaller res, and the rest for everything else. 

Can you remake these with a transparent background, instead of a white one?

**Comment by RichardLitt** [comment]
Also, of course, great work. :)

**Comment by Erioldoesdesign** [comment]
Doh! I didn't even notice the white background. Changed the files and removed white bg: https://drive.google.com/drive/folders/10ZI4FgZfh1h2LTxwboNPQN0hyb7_PSi9?usp=sharing

**Comment by Erioldoesdesign** [comment]
Good to close? @RichardLitt

**Comment by RichardLitt** [comment]
I'd say no, at the moment. Changed the favicon, now. However, I'm unsure on the latest headers needed for all of the variants. What tool did you use to generate them? Any chance it gives guidance on the correct html tag for each?

**Comment by RichardLitt** [comment]
That's a different issue. Making a new issue for it.

**Comment by Erioldoesdesign** [comment]
@RichardLitt Used this one: https://www.favicon-generator.org/
